build/ci/multiarch/helper_to_push_csi_images_and_manifests.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash -ex
+
+# ----------------------------------------------------------------
+# This script is for internal use in CI
+# The script push all csi images from internal registry to external registry.
+# Images for amd64, ppc64le and s390x for each csi image : operator, controller and node.
+# It also creates and pushes relevant manifests per architecture into the external repository.
+# The script validates the whole process. If something gets wrong the script will fail with error.
+# Pre-requisites:
+#    1. Run docker login to the external registry in advance.
+#    2. The internal images should be exist in advance.
+#    3. The external images should NOT be exist (the script will creates them).
+#    4. Helper scripts should be accessible: ./helper_to_push_docker_image.sh and ./helper_to_push_docker_manifest.sh
+#    5. Scripts input comes from environment variables
+# ----------------------------------------------------------------
+
+function push_arch_images_and_create_manifest_for_app() {
+  # This function push <app> arch images(X, P and Z) from internal registry to external registry and then create manifest for them on the external registry.
+  # Function arguments :
+  #   <app name>
+  #   <in_app_image_AMD64>   <out_app_image_AMD64>
+  #   <in_app_image_PPC64LE> <out_app_image_PPC64LE>
+  #   <in_app_image_S390X>   <out_app_image_S390X>
+  #   <out_app_image_MULTIARCH>
+  #   <tag_LATEST>
+
+  app_name="$1"
+  in_app_image_AMD64="$2"
+  out_app_image_AMD64="$3"
+  in_app_image_PPC64LE="$4"
```

Comment:
do we need ppc64? (is it power?)

Reply:
it power, yes

Reply:
A.K.A [PowerPC](https://en.wikipedia.org/wiki/PowerPC#Operating_systems_with_native_support)

Reply:
I think one of you should resolve this